### DISCUSSION TITLE
how to fix idealTree:lib: sill idealTree buildDeps

### DISCUSSION THREAD

**Discussion by qqwqqw689** [hexojs/hexo#5201] [Question / Answer] [answered]
<img width="321" alt="eb95bad22074111c3d993b7ce6ca7b5" src="https://user-images.githubusercontent.com/114795525/236632238-a469b031-40fc-472e-8614-b5efe9f8c16a.png">

**Comment by yoshinorin** [comment, marked as answer]
@qqwqqw689 

I assume this issue caused by network problem.
Would you please check your network condition? e.g) proxy, firewall, etc...

**Reply by qqwqqw689** [reply]
Thanks a lot,I have already fix it,It is because I use VPN to bypass the firewall.
